back/engines/free/email_campaigns/config/locales/en.yml

```diff
@@ -419,13 +419,15 @@ en:
       cta_view_phase: 'Discover this new phase'
       event_description: 'This project entered a new phase on the platform of %{organizationName}. Click on the link below to learn more!'
       main_header: 'A new phase started for project ''%{projectName}'''
+      subtitle: 'About ''%{projectName}'''
```

Comment:
I'm assuming that making these changes will cause Crowdin to update to include these new translations, correct?

Reply:
If you've added this branch to Crowdin, then I assume yes. But make sure the translations are done before merging according to this framework https://www.notion.so/citizenlab/Translations-e664aac66cdc42cc84e741fcf7d585b7#e39db377ea80473088a8b1eec61af579

Reply:
@alexander-cit Where in that document does it talk about making sure the translations are done? 

> make sure the translations are done before merging

Before merging to master? Or to production?

Reply:
> Where in that document does it talk about making sure the translations are done?

> we never release with any missing translations

> Before merging to master? Or to production?

Before master, because sbd else can release to production.

Reply:
> If you've added this branch to Crowdin, then I assume yes.

I've just added the branch, and started 'sync' on Crowdin GH integration.